nuxeo-features/nuxeo-platform-csv-export/src/test/java/org/nuxeo/ecm/platform/csv/export/TestCSVExportAction.java

```diff
@@ -194,14 +196,21 @@ public void testExportWithParams() throws Exception {
                 child.setPropertyValue("dc:subjects", new String[] { "art/architecture" });
                 testIds.add(child.getId());
             }
+            if (child.getType().equals("File")) {
+                Blob blob = Blobs.createBlob("the blob content");
+                blob.setFilename("initial_name.txt");
+                child.setPropertyValue("file:content", (Serializable) blob);
+            }
+
```

Comment:
We should also add the case if we set a null blob to test what was added in `DocumentPropertyCSVWriter`